values.yaml

```diff
@@ -168,6 +168,12 @@ gateway:
     port: 443
     externalTrafficPolicy: Local
     annotations: {}
+  api:
+    containerPort: 9696
+    service:
+      enabled: false
+      type: NodePort
+      port: 9696
```

Comment:
This should default to the default for gateway, which is that it listens on the same port that gateway does.

Reply:
Based on the other comment, this can be ignored.. Seems like the chart already assumes 9696 in other places.